pages/how-to/set-up/new-mgmt-cluster.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Setting Up a New MGMT Cluster
+description: Using Plural CLI to Deploy a MGMT Kubernetes Cluster
+---
+
+### Prerequisites
+[Plural CLI](/how-to/set-up/plural-cli)
+
+##### Ensure Cloud Provider CLI Authentication
+Plural uses the _default_ profile when deploying resources  
+
+AWS  
+```sh
+aws sts get-caller-identity
+```  
+AZ
+```sh
+az account show
+```
+GCP
+```sh
+gcloud auth list
+```
+
+
+### Create a New Repo for Plural
```

Comment:
We shouldn't recommend they do this, instead let `plural up` create the repo for them, which it does via oauth (and explain it can be cloned first if they can't do that due to permissioning constraints).